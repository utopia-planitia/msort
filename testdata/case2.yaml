
klm: nop
# comment
ghi: jkl
abc: def
abcd: defd

---

foo: fara
bar: 1
