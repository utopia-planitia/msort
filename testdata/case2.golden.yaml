abc: def
abcd: defd
# comment
ghi: jkl
klm: nop
---
bar: 1
foo: fara
